config/locales/address/en_BD.yml

```diff
@@ -0,0 +1,11 @@
+en_BD:
+  helpers:
```

Comment:
Why not use https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models?

Reply:
To avoid using `Model.human_attribute_name(attribute)`. Since our current requirement is literally only labels, we can add this when we start using the labels throughout the codebase.